conversation.py

```diff
@@ -14,9 +14,10 @@ def __init__(self, game, engine, xhr, version, challenge_queue):
     command_prefix = "!"
 
     def react(self, line, game):
-        logger.info(f'*** {self.game.url()} [{line.room}] {line.username}: {line.text.encode("utf-8")}')
-        if line.text[0] == self.command_prefix:
-            self.command(line, game, line.text[1:].lower())
+        if line.username != self.game.username:
```

Comment:
Won't the bots be unable to respond to themselves with this line? I think that if a bot owner types !eval, their bot should be able to respond.